fedimint-core/src/session_outcome.rs

```diff
@@ -15,11 +15,16 @@ pub struct AcceptedItem {
     pub peer: PeerId,
 }
 
-/// All items ordered in a session that have been accepted by Fedimint Consensus
-/// are recorded in the corresponding block. A running Federation produces a
-/// [SessionOutcome] roughly every five minutes.  Therefore, just like in
-/// Bitcoin, a [SessionOutcome] might be empty if no items are ordered in that
-/// time or all ordered items are discarded by Fedimint Consensus.
+/// Items ordered in a single session that have been accepted by Fedimint
+/// consensus.
+///
+/// A running Federation produces a [SessionOutcome] roughly every five minutes.
+/// Therefore, just like in Bitcoin, a [SessionOutcome] might be empty if no
+/// items are ordered in that time or all ordered items are discarded by
+/// Fedimint Consensus.
+///
+/// When session is closed, signed over by the peers, producing
+/// [`SignedSessionOutcome`]
```

Comment:
Perhaps this was meant to be combined with the docs added for `SignedSessionOutcome`?

Reply:
Just reread, scratch my previous comment.

It might be an improvement to update `producing` to `it produces a`.